inference-engine/thirdparty/clDNN/src/gpu/loop_gpu.cpp

```diff
@@ -0,0 +1,399 @@
+/*
+// Copyright (c) 2021 Intel Corporation
```

Comment:
This copyright message format is outdated. Please use new version.

Reply:
This will be applied all added files

Reply:
All outdated copyright message are fixed